RELEASES.md

```diff
@@ -79,4 +88,85 @@ deadline of 3 weeks.
 | 1.22.0  | 2022/04/15 | 2022/04/15 |    0 days  | 2023/04/15  |
 | 1.23.0  | 2022/07/15 |            |            |             |
 
-[repokitteh]: https://github.com/repokitteh
+### Cutting a major release
+
+* Take a look at open issues tagged with the current release, by
+  [searching](https://github.com/envoyproxy/envoy/issues) for
+  "is:open is:issue milestone:[current milestone]" and either hold off until
+  they are fixed or bump them to the next milestone.
+* Begin marshalling the ongoing PR flow in this repo. Ask maintainers to hold off merging any
+  particularly risky PRs until after the release is tagged. This is because we aim for main to be
+  at release candidate quality at all times.
+* Do a final check of the [release notes](docs/root/version_history/current.rst):
```

Comment:
no - not here - this branch wants to keep links to its own version - we only update the `ref:` links _after_ a release

in the case of a release from `main` (described as "major release" here) we want to update the `ref:` links for the `v1.X.Y.rst` files after the release has happened

this is because `main` is now on a new (pending) version so the just released version has now become historical - any future branches will also carry the updated `ref:` s

to use a concrete example - at the point of just happened `v1.22.0`:

- when release happens `current.rst` has links with _no_ version prefix
- after release `v1.22.0` is now historical so all links need to be updated _on `main`_
- on the `release/v1.22` branch (and patches cut from it)  the links from all `v1.22.x.rst` files will always _not_ have the link prefix as the version is contemporaneous

Reply:
@phlax Can you suggest text to be added here?